docs/source/scilla-in-depth.rst

```diff
@@ -2226,15 +2228,14 @@ init.json should have an entry:
 
 Namespaces
 **********
-When a contract or a library imports more than one library, a possibility of name clash
-exists. For example if library ``X`` and library ``Y`` are imported, and they both have
-a name ``foo`` defined, then Scilla cannot decide which ``foo`` it must resolve
-to when ``foo`` is used in the importer. The checker flags an error in such a scenario.
+Import statements can be used to define separate namespaces for imported names.
+To push the names from a library ``Foo`` into the namespace ``Bar``, use the statement
+``import Foo as Bar``. Accessing a variable ``v`` in Foo must now be done using the qualified
+name ``Bar.v``. This is useful when importing multiple libraries that define the same name.
 
-To enable users to meaningfully import libraries that may have name clashes, Scilla provides
-an ``import as`` construct. ``import Foo as Bar`` imports the library ``Foo`` with all of its
-names pushed into the ``Bar`` namespace. Accesses to names in ``Foo`` must now be prefixed
-with ``Bar.``.
+The same variable name must not occur in the same namespace, so if multiple imported libraries
```

Comment:
"The same variable name must not occur in the same namespace,..." ->
"The same variable name must not be defined more than once in the same namespace, ..."

I'm surprised neither of us caught the nonsense of that sentence. :-)

Reply:
ha ha, you're right.